draft-ietf-quic-transport.md

```diff
@@ -1921,9 +1921,13 @@ that contain packets that are successfully processed and datagrams that contain
 packets that are all discarded.
 
 Clients MUST ensure that UDP datagrams containing Initial packets have UDP
-payloads of at least 1200 bytes, adding PADDING frames as necessary.
-A client that sends padded datagrams allows the server to
-send more data prior to completing address validation.
+payloads of at least 1200 bytes, adding PADDING frames as necessary.  Servers
+MUST ensure that UDP datagrams containing ack-eliciting Initial packets have UDP
+payloads of at least 1200 bytes.  A client that sends padded datagrams allows
+the server to send more data prior to completing address validation.  Endpoints
+sending UDP datagrams of at least 1200 bytes ensures that the handshake
+progresses only if the path is capable of handling QUIC traffic; see
+{{packet-size}}.
```

Comment:
```suggestion
payloads of at least 1200 bytes, adding PADDING frames as necessary.  Servers
MUST ensure that UDP datagrams containing ack-eliciting Initial packets have UDP
payloads of at least 1200 bytes. Sending UDP datagrams of at least 1200 bytes
ensures that the handshake progresses only if the path is capable of handling
QUIC traffic; see {{packet-size}}. Additionally, a client that sends padded
datagrams allows the server to send more data prior to completing address
validation.
```

Reply:
In addition, let's remove the server text from here, since that is unnecessary for address validation. The server datagram padding needs to only be mentioned below, in the Initial Packet Size section.

Reply:
@janaiyengar Following your last comment, I've reverted the changes to this section (i.e. address validation). That means that the suggested change right above is not applied. Please see if you want anything to be resurrected.